module/proto/gravity/v1/query.proto

```diff
@@ -110,10 +110,15 @@ service Query {
     option (google.api.http).get =
         "/gravity/v1/cosmos_originated/denom_to_erc20";
   }
-  // Query for pending tranfertxs
-  rpc PendingSendToEthereums(PendingSendToEthereumsRequest)
-      returns (PendingSendToEthereumsResponse) {
-    option (google.api.http).get = "/gravity/v1/query_pending_send_to_eth";
+  // Query for batch send to ethereums
```

Comment:
q: why the plural for ethereum***s*** ?

Reply:
They are sends to ethereum but because the struct is `SendToEthereum` the plural would be `SendToEtheuerms`